tensorflow/compiler/xla/service/platform_util.cc

```diff
@@ -48,9 +51,14 @@ string CanonicalPlatformName(const string& name) {
   if (platform_str == "cpu") {
```

Comment:
Since you are changing this function, could we have different variable names for input and output?